lecture_02_data_structures_functions/hw/hw2.py

```diff
@@ -0,0 +1,22 @@
+"""
+Given an array of size n, find the max and min element.
+The max element is the element that appears more than n // times.
+The min element is the element that appears fewer than other.
+
+You may assume that the array is non-empty and the max element always exist in the array.
+
+Example 1:
+
+Input: [3,2,3]
+Output: 3, 2
+Example 2:
+
+Input: [2,2,1,1,1,2,2]
+Output: 2, 1
+
+"""
+from typing import List, Tuple
+
+
+def major_and_minor_elem(inp: List) -> Tuple[int, 2]:
```

Comment:
```suggestion
def major_and_minor_elem(inp: List[int]) -> Tuple[int, int]:
```
https://docs.python.org/3/library/typing.html#typing.Tuple